NineMensMorris/src/singleplayerscreen.cpp

```diff
@@ -0,0 +1,36 @@
+#include "include/singleplayerscreen.h"
+
+SinglePlayerScreen::SinglePlayerScreen(QGraphicsScene *scene) {
+    this->scene = scene;
+    displaySinglePlayerScreen();
+}
+
+void SinglePlayerScreen::displaySinglePlayerScreen() {
+
+    //Instantiate items
+    QFont buttonFont("comic sans MS", 14);
+    whiteButton = new QPushButton(QString("White"),NULL);
+    blackButton = new QPushButton(QString("Black"),NULL);
+
+    //declare the choose color text
+    QGraphicsTextItem *chooseColor = scene->addText("Choose the piece color to play");
```

Comment:
```suggestion
    QGraphicsTextItem *chooseColor = scene->addText("Choose your color");
```
I think making the prompt simpler like above would help with readability